modules/lib/lib-content/src/main/resources/lib/xp/content.js

```diff
@@ -8,6 +8,10 @@
  */
 /* global __*/
 
+exports.CONTENT_ROOT_PATH = com.enonic.xp.content.ContentConstants.CONTENT_ROOT_PATH;
```

Comment:
I don't like that approach I would prefer to use helper for constants. But it works so I approve pr.